Xamarin.Forms.Platform.Android/CellAdapter.cs

```diff
@@ -334,5 +338,12 @@ bool OnPrepareActionModeImpl(IMenu menu)
 
 			return false;
 		}
+
+
+		protected override void Dispose(bool disposing)
```

Comment:
Since this doesn't actually do anything in the Dispose override, do we need it? I realize it's checking `_isDisposed`, but doesn't the `IsDisposed()` extension cover that?

Reply:
I wasn't sure if it did. IsDisposed() checks the Handle which I wasn't sure if that gets synchronously set to Zero when Dispose is called or not. But I checked with @jonathanpeppers and it looks like just checking if the handle is Zero is the same thing

https://github.com/xamarin/xamarin-android/blob/75e053e5eff431093c7e00d94962fcd13683be5c/src/Mono.Android/Java.Lang/Object.cs#L201-L241

Reply:
Yeah, the `IsDisposed` method was just added to make the intent more explicit; otherwise we end up with tons of `Handle == IntPtr.Zero` checks sprinkled about.